src/notification/apns.ts

```diff
@@ -0,0 +1,40 @@
+import {Link, Store} from '../store/model';
+import {Print, logger} from '../logger';
+import * as apn from 'apn';
+import {config} from '../config';
+
+const { apns } = config.notifications;
+
+export function sendApns(link: Link, store: Store) {
+  var options = {
```

Comment:
```suggestion
  const options = {
```

Do you mind changing this to `const` and a couple variables that don't change below?

Reply:
Done